portal/tasks.py

```diff
@@ -269,6 +273,56 @@ def send_user_messages(email, force_update=False):
     return message
 
 
+@celery.task
+def send_questionnaire_summary(job_id, cutoff_days, org):
+    "Generate and send a summary of questionnaire counts to all Staff in org"
+    try:
+        before = datetime.now()
+        generate_and_send_summaries(cutoff_days, org)
+        duration = datetime.now() - before
+        message = (
+            'Sent summary emails in {0.seconds} seconds'.format(duration))
+        current_app.logger.debug(message)
+    except Exception as exc:
+        message = ("Unexpected exception in `send_questionnaire_summary` "
+                   "on {} : {}".format(job_id, exc))
+        logger.error(message)
+        logger.error(format_exc())
+    update_current_job(job_id, 'send_questionnaire_summary', status=message)
+    return message
+
+
+def generate_and_send_summaries(cutoff_days, org):
+    ostats = overdue_stats_by_org()
+    cutoffs = [int(i) for i in cutoff_days.split(',')]
+
+    ot = OrgTree()
+    top_org = Organization.query.filter_by(name=org).first()
+    if not top_org:
+        raise ValueError("No org with name {} found.".format(top_org))
+
+    if top_org:
```

Comment:
Won't this always be `True` (otherwise above `ValueError` will be thrown)?

Reply:
Yep, good point